lib/home_screen/view/home_screen.dart

```diff
@@ -607,14 +615,26 @@ class _BottomNavBarState extends State<BottomNavBar> {
               enableFeedback: false,
               onPressed: () {
                 setState(() {
```

Comment:
Dont use setState here, it will cause unneeded rebuilds of your page, just call widget.onChanged() and that will suffice